File: JAVA Programming/BubbleSort.java
public class BubbleSort {

    public static void Bubble_Sort(int A[], int n) {
        int i, j;
        for (i = 0; i < n; i++) {
            for (j = 0; j < n - 1; j++) {
                if (A[j] > A[j + 1]) {
                    int temp = A[j];
                    A[j] = A[j + 1];
                    A[j + 1] = temp;
                }
            }
        }
    }

    public static void main(String[] args) {
        int arr[] = { 4, 7, 9, 21, 3, 4, 5, 1 }; // size 8 ;
        int n = 8;
        System.out.println("Print Array Before Sorting ");
        for (int e : arr) {

            System.out.print("\t" + e);

        }

        System.out.println("\nPrint Array After Sorting ");
        Bubble_Sort(arr, n);
        for (int e : arr) {

            System.out.print("\t" + e);

        }
    }

}

File: C++ Programming/Bubble_sort.cpp
#include<iostream>
using namespace std;
void Print(int* arr,int size)
{
    int i=0;
    cout<<"The array is ---- \n";
    for(i=0;i<size;i++)
    cout<<arr[i]<<"\t";
    cout<<endl<<endl;
}
void Boublesort(int* arr,int size)
{
    int i,j,temp;
    for(i=0;i<size;i++)
    {
        for(j=0;j<size-1-i;j++)
        {
            if(arr[j] > arr[j+1])
            {
                temp = arr[j];
                arr[j] = arr[j+1];
                arr[j+1] = temp;
            }
        }
    }
}
int main(int argc, char const *argv[])
{
    int size = 10;
    int arr[size] = {1,56,23,12,5,67,321,567,53,20,37};
    Print(arr,size);
    cout<<"After the Selection sort \n";
    Boublesort(arr,size);
    Print(arr,size);
    return 0;
}
